lib/generate-sitemap.js

```diff
@@ -16,18 +16,17 @@ function generateSitemap(batfishConfig) {
     path.join(batfishConfig.outputDirectory, 'sitemap.xml')
   );
   return new Promise((resolve, reject) => {
-    sitemapStatic(
-      sitemapWriter,
-      {
+    try {
```

Comment:
The only reason I had in mind was strict control over the value with which the Promise will resolve. With `() => resolve()` we can say that this function returns a Promise that resolves with `undefined`. With just `resolve`, I'm not exactly sure what it will resolve with — would have to check the Node docs — and I'm not sure it'd be valuable to the consumer of this function.